### PR TITLE
Fix link in README

### PR DESCRIPTION
**Pull request by pitmonticone**
Just fixed the link to your website https://mlcolab.org/sbi-workshop.

Thank you very much for making all this great material publicly available for free!

### CODE REVIEW

**Comment by pitmonticone** [comment]
CC: @sethaxen @jan-matthis

File: slides/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Workshop Slides
 
-Directory for slides of the [Simulation-based Inference workshop](mlcolab.org/sbi-workshop). It contains an example for `jupyter` slides with interactive elements. We strive to use these text-based slides as it makes further development simpler! 
+Directory for slides of the [Simulation-based Inference workshop](https://mlcolab.org/sbi-workshop). It contains an example for `jupyter` slides with interactive elements. We strive to use these text-based slides as it makes further development simpler!
```
Comment (sethaxen) [inline slides/README.md]: Thanks for noticing the dead URL! Actually, it seems even with the `https://`, the URL is dead. This should point to the correct website: 

```suggestion
Directory for slides of the [Simulation-based Inference workshop](https://mlcolab.org/resources/simulation-based-inference-for-scientific-discovery). It contains an example for `jupyter` slides with interactive elements. We strive to use these text-based slides as it makes further development simpler! 
```

**Review by sethaxen — CHANGES_REQUESTED** [review]

File: slides/README.md (hunk 1 above)
Comment (pitmonticone) [reply slides/README.md]: Thank you @sethaxen for the correction.

**Review by sethaxen — APPROVED** [review]
LGTM, thanks!

**Pull request merged**